bpf/bpf_lxc.c

```diff
@@ -562,7 +562,12 @@ __section_tail(CILIUM_MAP_CALLS, CILIUM_CALL_ARP) int tail_handle_arp(struct __s
 __section("from-container")
 int handle_ingress(struct __sk_buff *skb)
 {
-	int ret;
+	int ret = 0;
+	skb->cb[CB_SRC_LABEL] = ret;
+	skb->cb[CB_IFINDEX] = ret;
+	skb->cb[CB_POLICY] = ret;
+	skb->cb[CB_NAT46_STATE] = ret;
+	skb->cb[CB_CT_STATE] = ret;c
 
```

Comment:
May be we should add an inline function or macro to common.h instead of repeating this code in 4 places.